test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -251,6 +251,23 @@ test('clean up outerHTML', function () {
     }
 });
 
+asyncTest('xhr.responseURL', function () {
+    var xhr   = new XMLHttpRequest();
+
+    xhr.addEventListener('readystatechange', function () {
+        if (this.readyState === XMLHttpRequest.DONE) {
+            if (this.responseURL)
+                strictEqual(eval(processScript('this.responseURL')), 'https://example.com/xhr-test/');
```

Comment:
More simple:
`strictEqual(getProperty('this.responseURL')), 'https://example.com/xhr-test/');`